### PR TITLE
Gun Game: Fix potentially transferring score from leaving player to new joining player

### PR DESCRIPTION
**Pull request by Interesting-exe**
fixed a bug in which a player that just joined the server might have someone who's left the server's score
(this fix should be applied to all ffa gamemodes)

### CODE REVIEW

**Comment by lnteresting-png** [comment]
as mentioned earlier the fix should be added to all ffa gamemodes im just too lazy to do it

**Comment by BobTheBob9** [comment]
why not just do `AddTeamScore( player, -GameRules_GetTeamScore( player.GetTeam() ) )`

**Comment by lnteresting-png** [comment]
> why not just do `AddTeamScore( player, -GameRules_GetTeamScore( player.GetTeam() ) )`

i just copy and pasted the bankrupt function from the sns gamemode

**Comment by GeckoEidechse** [comment]
@lnteresting-png I updated the PR title to better reflect the intended fix of the PR. If I didn't get it right, please LMK.

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_gg.gnut [hunk 1]
```diff
@@ -27,6 +28,12 @@ void function GamemodeGG_Init()
 	catch ( ex ) {}
 }
 
+void function OnPlayerDisconnected(entity player)
+{
+	// resetting a player's score when he leaves prevents a new player to join the leaver's team and having his points
```
Comment (GeckoEidechse) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_gg.gnut:33]:
```suggestion
	// resetting a player's score when they leave prevents a new player to join the leaver's team and having the previous player's points
```

**Comment by lnteresting-png** [comment]
> @lnteresting-png I updated the PR title to better reflect the intended fix of the PR. If I didn't get it right, please LMK.

yeah you got it right

**Comment by x3Karma** [comment]
yeah i seen it happening in sticks and stones as well,

**Comment by BobTheBob9** [comment]
looks great, if you add this to ffa i'll merge

**Comment by BobTheBob9** [comment]
will add ffa support myself later

**Pull request merged**